docs/example/workspace.dsl

```diff
@@ -170,6 +170,9 @@ workspace "Big Bank plc" "This is an example workspace to illustrate the key fea
             // * it's not possible to use "GitLab" and "ResizableImage" extensions together
             // default behaviour, if no generatr.markdown.flexmark.extensions property is specified, is to load the Tables extension only
             "generatr.markdown.flexmark.extensions" "Abbreviation,Admonition,AnchorLink,Attributes,Autolink,Definition,Emoji,Footnotes,GfmTaskList,GitLab,MediaTags,Tables,TableOfContents,Typographic"
+
+             // default behaviour, if no is generatr.markdown.mermaid.enabled property i specified, is "true"
+            "generatr.markdown.mermaid.enabled" "true"
```

Comment:
Is this setting really necessary? Are there scenario's where someone would prefer the outdated GitLab plugin behaviour? If not, I would prefer that the workaround is applied automatically if the `GitLab` FlexMark plugin is enabled.

Reply:
The PR uses a new implementation for integrating mermaid diagrams in a markdown file, that works independently from the GitLab extension. If you would like, to have this feature always available, than we may omit the separate flag. If mermaid support should be an optional feature, than we need a separate flag, because it is not neccessary to enable the GitLab extension if somebody wants to use mermaid diagrams. If somebody does not want to use all features included in GitLab Flavored Markdown (GLFM), but needs mermaid support, than this is possible with the current PR implementation.

Reply:
Ah, I see, thanks for the clarification! I don't think there's a clear downside to having this Mermaid support always enabled, so I'd suggest removing this option.

@jp7677 Any thoughts on this?

Reply:
I haven't played yet with all the fancy new markdown extensions, so take my advice with care. But from what I read here, I would also prefer to avoid the configuration option and enable mermaid by default. If that conflicts with another flexmark extension, I would rather blacklist the conflicting extension, esp. if it is somewhat  outdated anyway.

Reply:
OK, i'll remove the code, that is needed, to make mermaid support optional.